src/mysql.c

```diff
@@ -995,6 +1024,54 @@ static int mysql_read (user_data_t *ud)
 		{
 			counter_submit ("mysql_slow_queries", NULL , val, db);
 		}
+ 		else if (strncmp (key, "Queries", strlen ("Queries")) == 0) 
+ 		{
+ 			counter_submit ("mysql_queries", NULL , val, db);
+ 		}
+ 		else if (strncmp (key, "Connections", strlen ("Connections")) == 0) 
+ 		{
+ 			counter_submit ("mysql_connections", NULL , val, db);
+ 		}
+ 		else if (strncmp (key, "Uptime", strlen ("Uptime")) == 0) 
+ 		{
+ 			counter_submit ("mysql_uptime", NULL , val, db);
+ 		}
+ 		else if (strncmp (key, "Questions", strlen ("Questions")) == 0) 
+ 		{
+ 			counter_submit ("mysql_questions", NULL , val, db);
+ 		}
+ 		else if (strncmp (key, "Created_", strlen ("Created_")) == 0)
+                 {
+                        if (strcmp (key, "Created_tmp_tables") == 0)
+                                counter_submit ("mysql_created", "tmp_tables", val, db);
+                        else if (strcmp (key, "Created_tmp_disk_tables") == 0)
+                                counter_submit ("mysql_created", "tmp_disk_tables", val, db);
+                        else if (strcmp (key, "Created_tmp_files") == 0)
+                                counter_submit ("mysql_created", "tmp_files", val, db);
+                 }
+ 		else if (strncmp (key, "Aborted_", strlen ("Aborted_")) == 0)
+ 		{
+ 			if (strcmp (key, "Aborted_clients") == 0)
+ 				counter_submit ("mysql_aborted", "clients", val, db);
+ 			else if (strcmp (key, "Aborted_connects") == 0)
+ 				counter_submit ("mysql_aborted", "connects", val, db);
```

Comment:
@octo, I think total_conections should be reviewed in complex.

We put 'total_connections -> aborted-connects" here and "total_connections -> aborted-clients" upper. Full list of  type instances of this tupe should be compiled and checked.

And mix of 'connects' and 'clients' does not look nice.